lingua_franca/res/text/en-us/normalize.json

```diff
@@ -99,7 +99,78 @@
     "you'd've": "you would have",
     "you'll": "you will",
     "you're": "you are",
-    "you've": "you have"
+    "you've": "you have",
+    "I'm'a": "I am about to",
+    "I'm'o": "I am going to",
+    "I'll've": "I will have",
+    "I'd've": "I would have",
+    "Whatcha": "What are you",
+    "amn't": "am not",
+    "'cause": "because",
+    "can't've": "cannot have",
+    "couldn't've": "could not have",
+    "daren't": "dare not",
+    "daresn't": "dare not",
+    "dasn't": "dare not",
+    "everyone's": "everyone is",
+    "gimme": "give me",
+    "gon't": "go not",
+    "hadn't've": "had not have",
+    "he've": "he have",
```

Comment:
Not sure about this one; I'm not sure where "he have" is a valid combination of words (he has? he would have?)

Reply:
so what is the change to make here?

- he has
- he would have
- remove entry